wgpu/Cargo.toml

```diff
@@ -25,33 +25,78 @@ targets = [
 
 [features]
 default = ["wgsl", "dx12", "metal"]
-# Apply run-time checks, even in release builds. These are in addition
-# to the validation carried out at public APIs in all builds.
-strict_asserts = ["wgc?/strict_asserts", "wgt/strict_asserts"]
+
+#! ### Backends
+# --------------------------------------------------------------------
+#! ⚠️ WIP: Not all backends can be manually configured today.
+#! On Windows & Linux the Vulkan & GLES backends are always enabled.
+#! See [#3514](https://github.com/gfx-rs/wgpu/issues/3514) for more details.
+
+## Enables the DX12 backend on Windows.
+dx12 = ["wgc?/dx12"]
+
+## Enables the Metal backend on macOS & iOS.
+metal = ["wgc?/metal"]
+
+## Enables the GLES backend via [ANGLE](https://github.com/google/angle) on Windows & macOS using.
+angle = ["wgc?/gles"]
+
+## Enables the Vulkan backend on macOS & iOS.
+vulkan-portability = ["wgc?/vulkan"]
+
+## Enables the GLES backend on Wasm
+##
+## * ⚠️ WIP: Currently will also enable GLES dependencies on any other targets.
+## * ⚠️ WIP: This automatically disables the WebGPU feature. See [#2804](https://github.com/gfx-rs/wgpu/issues/3514).
+webgl = ["hal", "wgc/gles"]
+
+#! ### Shading language support
+# --------------------------------------------------------------------
+#! The following features do not have any effect on the WebGPU backend.
```

Comment:
```suggestion
```

They enable Naga to transpile on the Webgpu backend.

Reply:
ohhh I didn't realize. Super cool.
The best way to learn something is to write a PR with the wrong documentation :D

Reply:
guess then this doesn't need much extra clarification at all. Seems a bit convoluted to go into the details of what drags in what dependency when the gist is that each of these features enable the given shading language no matter what :)